CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # @digitalbazaar/cborld ChangeLog
 
+## 7.2.0 - 2024-10-dd
+
+### Fixed
+- Fix varint processing when registry IDs require multiple bytes.
+
+### Changed
+- **BREAKING**: When a registry entry is used that requires more
+  than one byte, the payload is now a two element tagged array
+  containing a bytestring and the encoded JSON-LD document
+  instead of a sequence containing a tagged bytestring and
+  the encoded document.
```

Comment:
```suggestion
- Fix bug with registry IDs that are expressed using varints larger than
  1 byte in size. This would break any previous use of this, but that
  previous use is invalid and non-interoperable. When a registry entry is
  used that requires more than one byte, the payload is now appropriately
  a two element tagged array containing a bytestring and the encoded
  JSON-LD document instead of a sequence containing a tagged
  bytestring and the encoded document.
```